doc/whats_new/v0.21.rst

```diff
@@ -351,6 +353,12 @@ Support for Python 3.4 and below has been officially dropped.
 - |Feature| Allow different dtypes (such as float32) in
   :class:`isotonic.IsotonicRegression` :issue:`8769` by :user:`Vlad Niculae <vene>`
 
+:mod:`sklearn.utils`
+....................
+- |Fix| Fixed an issue in :func:`utils.multiclass._ovr_decision_function` which would give
+  different results when applied to a subset due to local scaling method.
+  :pr:`10440` by :user:`Jonathan Ohayon <Johayon>`
```

Comment:
you should document the public funcions/classes impacted by this PR not the private functions. These are not part of the documentation.

Reply:
ok, is it better like this ?